docs/Running.rst

```diff
@@ -0,0 +1,37 @@
+.. _running:
+
+Running
+=======
+
+Note that instructions on how to write an input file can be found
+in the :ref:`Input Manual<input-manual>`.
+
+Running SCONE
+'''''''''''''
+
+After installation of all the dependencies and the compilation of SCONE, 
+one can finally run the code. The executable ``scone.out`` is in the ``build``
+folder; one can run it from any folder by specifying the path to the executables
+and to the input file::
```

Comment:
Perhaps it would be good to clearly describe the interaction between OpenMP and MPI?